src/test/java/org/minions/devfund/marylin/MovieDatabaseTest.java

```diff
@@ -0,0 +1,91 @@
+package org.minions.devfund.marylin;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Movie database test class.
+ */
+public class MovieDatabaseTest {
+    private MovieDatabase movieDatabase;
+
+    /**
+     * Method to create an instance of CC class.
+     */
+    @Before
+    public void setUp() {
+        final double rating1 = 50.0;
+        final double rating2 = 25.0;
+        final double rating3 = 35.0;
+        final String movie1 = "Sleepers";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "Sleepers" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)